core/trino-main/src/main/java/io/trino/execution/buffer/PartitionedOutputBuffer.java

```diff
@@ -94,6 +95,12 @@ public double getUtilization()
         return memoryManager.getUtilization();
     }
 
+    @Override
+    public TDigest getUtilizationDigest()
+    {
+        return memoryManager.getUtilizationDigest();
```

Comment:
@radek-starburst you are right that this stat should be part of `OutputBufferInfo`. However, we can still summarize it in `StageStats` (see method `io.trino.execution.StageStateMachine#getStageInfo`)